tools/accuracy_checker/accuracy_checker/launcher/tf_launcher.py

```diff
@@ -1,8 +1,25 @@
+"""
+Copyright (c) 2019 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
 import tensorflow as tf
 
 from .launcher import Launcher
 from ..config import BaseField, ListField, PathField, StringField, ConfigError, ConfigValidator
 
+
```

Comment:
@eaidova why extra new line added?

Reply:
It is standard code style to separate classes form import statements using 2 lines (instead 1 used in this file). It was applied by my IDE automatically.

Reply:
ok, this is python coding style, accepted